### PR TITLE
Reviewing Lines Part 1

### PR DESCRIPTION
**Pull request by ttgmichael**
Dolar signs on QUESTLV100 lines 0-700 with some revisions.

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -304,36 +304,36 @@ QUEST_LV_0100_20150317_000303	{memo X}$Thank you for making your way here.{nl}I
...
-QUEST_LV_0100_20150317_000330	$We might have to give up our current supplies and support ourselves until the next batch of supplies arrive.
+QUEST_LV_0100_20150317_000330	$We might have to give up our current supplies and support ourselves until the next batch of {nl}supplies arrive.
 QUEST_LV_0100_20150317_000331	$Thank you.{nl}With you helping us out like this, we will not be defeated.
 QUEST_LV_0100_20150317_000332	$Your next mission is to find the cause of the increasing number of monsters.{nl}Popolions have a big appetite, so I hope we can find a clue.
-QUEST_LV_0100_20150317_000333	$Although the Miners' Village is in trouble, given that the situation has reached the Eastern Woods,{nl}we can't assure the safety of Klaipeda.{nl}We can just hope that Klaipeda is doing well.
+QUEST_LV_0100_20150317_000333	$Although the Miners' Village is in trouble, given that the situation has reached the Eastern Woods,{nl}we can't assure the safety of Klaipeda.{nl}We hope the Miner's Village is putting up a good fight.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:333]: Miner's Village

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -382,44 +382,44 @@ QUEST_LV_0100_20150317_000381	{memo X}$The Miners' Village is under siege and en
 QUEST_LV_0100_20150317_000382	{memo X}$You have to cheer up.{nl} Even Aras is holding on.
 QUEST_LV_0100_20150317_000383	$Goddess Zemyna overlooks the Earth.{nl}There are rumors that the goddesses have abandoned us, but that's nonsense.{nl}The statues still shine beautifully when you pray to them.
 QUEST_LV_0100_20150317_000384	$Vubbes suddenly came pouring in into the Siauliai Woods from the Crystal Mines.{nl}Thus, the Miners' Village situated between the Mines and the Woods became a battlefield.{nl}
-QUEST_LV_0100_20150317_000385	$For safety reasons, I can't permit entrance to the Miners Village. {nl}Of course it would be a different story if your skill are good enough to ensure your own safety {nl}without the guards' protection.
+QUEST_LV_0100_20150317_000385	$For safety reasons, I can't permit entrance to the Miner's Village. {nl}Of course it would be a different story if your skill are good enough to ensure your own safety {nl}without the guards' protection.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:385]: Miner's Village

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -480,10 +480,10 @@ QUEST_LV_0100_20150317_000479	$I am tired of living a life running away from the
 QUEST_LV_0100_20150317_000480	$Thinking of the peaceful days in the past makes me feel like I'm dreaming.{nl}Being awake is like a very long nightmare.
 QUEST_LV_0100_20150317_000481	$I want to run away to another place if I can.{nl}Is there a future for the kingdom?
 QUEST_LV_0100_20150317_000482	$I can't sleep well nowadays.{nl}It's all because of those monsters.
-QUEST_LV_0100_20150317_000483	$To live at peace in a world where monsters are running wild... Impossible.
+QUEST_LV_0100_20150317_000483	$To live at peace in a world where monsters are running wild... is impossible.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:483]: Both versions work.

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -646,53 +646,53 @@ QUEST_LV_0100_20150317_000645	$Yes. My brother is a thief.{nl}My brother used to
...
 QUEST_LV_0100_20150317_000690	{memo X}It may be better for Gabija to let her know that we are here.{nl}There's a pole that we can ignite light on. Let's go.
 QUEST_LV_0100_20150317_000691	{memo X}Good. Since she knows that we came, she will recover her health.{nl}Let's go up to see Gabija.{nl}{nl}
 QUEST_LV_0100_20150317_000692	$Guardian of the Kingdom's Secret Treasures
-QUEST_LV_0100_20150317_000693	You are qualified to receive the secret treasures.
+QUEST_LV_0100_20150317_000693	$You are qualified to receive the secret treasures.
 QUEST_LV_0100_20150317_000694	{memo X}
-QUEST_LV_0100_20150317_000695	$The magic suppressor of the tomb can be destroyed using the activating stones found inside the Guardians.{nl}The suppressed magic of the tomb will unleash and suppress the vicious energy.
+QUEST_LV_0100_20150317_000695	$The magic regulator of the Mausoleum can be destroyed using the activating stones found{nl}inside the Guardians.The regulatory magic of the Mausoleum will then be released and will {nl}suppress the evil energy.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:695]: either: Royal Mausoleum or mausoleum (uncapitalized)

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -4771,24 +4771,24 @@ QUEST_LV_0100_20150317_004770	$The Guardian's Stone Statue told you that the mag
...
 QUEST_LV_0100_20150317_004777	$You were able to fix the flow of the magic, but there are still demons interfering with the flow of magic. Talk to the Guardian's Stone Statue.
-QUEST_LV_0100_20150317_004778	$Defeat the Guardians near the magic power containers
-QUEST_LV_0100_20150317_004779	$The Guardian's Stone Statue told you that the corrupted Guardians are trying to break the magic power containers. Defeat the Guardians near the magic power containers.
-QUEST_LV_0100_20150317_004780	You were able to protect the container by defeating the Guardians. Return to the Guardian's Stone Statue.
-QUEST_LV_0100_20150317_004781	{memo X}Defeat the Guardian who threatens to break the container
+QUEST_LV_0100_20150317_004778	$Defeat the Guardians near the Magic Vessels
+QUEST_LV_0100_20150317_004779	$The Guardian's Stone Statue told you that the corrupted Guardians are trying to break the Magic Vessels. Defeat the Guardians near the Magic Vessels.
+QUEST_LV_0100_20150317_004780	You were able to protect the Vessels by defeating the Guardians. Return to the Guardian's Stone Statue.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:4780]: Vessels -> vessels

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:483]: Yep, I'd rather him finish the sentence he started xD.

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**